elide-integration-tests/src/test/java/com/yahoo/elide/tests/AnyPolymorphismIT.java

```diff
@@ -0,0 +1,269 @@
+                .assertThat()
+                .statusCode(HttpStatus.SC_NO_CONTENT);
+
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get(propertyPath + sep + id)
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body(relationshipType, equalTo(smartphoneType));
+
+        //delete relation
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .body(prefix + "\", \"type\": \"property\", \"relationships\": {\"myStuff\": {\"data\": null}}}}")
+                .patch(propertyPath + sep + id)
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_NO_CONTENT);
+
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get(propertyPath + sep + id)
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("data.relationships.myStuff.data", equalTo(null));
+    }
+
+    @Test
+    public void testAnySubpaths() {
+        String id = RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .body(jsonParser.getJson(tractorAsPropertyFile))
+                .post(propertyPath)
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_CREATED)
+                .extract()
+                .path(idPath);
+
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get(propertyPath + "?page[totals]")
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("meta.page.totalRecords", greaterThan(0));
+
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get(propertyPath + sep + id + "/myStuff")
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("data.attributes.horsepower", equalTo(horsepower),
+                        idPath, equalTo(one));
+
+        //single entity so no page appropriate stuff
+        RestAssured
+                .given()
+                .contentType(JSONAPI_CONTENT_TYPE)
+                .accept(JSONAPI_CONTENT_TYPE)
+                .get(propertyPath + sep + id + "/myStuff?page[totals]")
+                .then()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("meta", equalTo(null));
```

Comment:
This doesn't seem right to me.  Why would meta be null?

Reply:
Because this relationship is a single entity so it doesn't paginate (a one to one), the pagination is ignored here.  I found this to be consistent with the rest of elide.